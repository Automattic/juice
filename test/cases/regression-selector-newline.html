<html><body>
<a href="#">Test</a>
<p>Test</p>
</body></html>
