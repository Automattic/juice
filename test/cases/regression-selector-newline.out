<html><body>
<a href="#" style="color: red;">Test</a>
<p style="color: red;">Test</p>
</body></html>
